### ISSUE TITLE
Keybase install error on MacOS

### ISSUE THREAD

**Issue by jithinrk** [keybase/client#11544]
Hi, I have been receiving this error message while doing a fresh install of Keybase on the Mac: "There was an error trying to run the install (2). Please run 'keybase log send' to report the error." The options Quit and Ignore are presented. Tapping ignore just leaves the GUI with the keybase logo and 'loading...'.

Also, it does look like the CLI tool isn't installed at the moment.

My log id: 4e01722c030895f43f19481c

Any help would be really appreciated.

**Comment by Firni11** [comment]
Hi, I'm struggling with the same problem as @jithinrk mentioned above.
While doing an install of Keybase an error window occurred. 
<img width="435" alt="screenshot" src="https://user-images.githubusercontent.com/23240859/44714993-f2424800-aab6-11e8-8387-d4f31ce06aec.png">
"Ignoring" this message leads to the following screen, which is loading endless.
<img width="800" alt="screenshot2" src="https://user-images.githubusercontent.com/23240859/44715036-156cf780-aab7-11e8-8fe7-28e834180948.png">
Furthermore the CLI tool doesn't work (also as mentioned above).

Operating System: macOS 10.13.6 High Sierra

Thanks for your help.

**Comment by maxtaco** [comment]
cc @strib

**Comment by strib** [comment]
@max why the cc?  The redirector isn't mentioned here.

@Firni11 if you run `keybase log send` and post the resulting log ID here, someone can look in more detail.

**Comment by Firni11** [comment]
@strib 
I tried, but "keybase: comand not found".

**Comment by Firni11** [comment]
can anyone help?

**Comment by strib** [comment]
It seems like you haven't fully installed the app.  Did you drag it into your Applications folder and launch it from there?

**Comment by jithinrk** [comment]
I have the same issue, and have been trying to run the Keybase app from the Applications folder (the Keybase command isn’t installed on my system either)

**Comment by strib** [comment]
I'm afraid I don't know anything about the mac command line installation.  @cjb do you happen to know why that wouldn't work?

**Comment by Firni11** [comment]
@strib
Yeah. I dragged the app into the application folder. From there I started the app and the window (which i mentioned above) popped out. Also all „keybase“ commands are impossible to run in the terminal. Terminal gives back:“command not found“.

**Comment by cjb** [comment]
Huh, can't find anything for that log ID.

My guess would be something like: you have /usr/local/bin already created, but it's owned by a different user, and your current user can't put /usr/local/bin/keybase into it.  Could that be it?

(Alternatively: you're out of disk space?)

**Comment by rridolfo** [comment]
I have the same issue -- has anyone resolved this on their own?

**Comment by maxtaco** [comment]
@rridolfo  please `keybase log send`

**Comment by rridolfo** [comment]
I've done this twice.   The resulting terminal output is: 

_MACC02VW06XHTDF:MacOs rich$ ./Keybase log send
No installer.json file
Version: 2.5.1-20180830164425+a18ce51fa9
2018-09-11 10:03:28.424 Keybase[3842:257927] *** WARNING: Textured window <AtomNSWindow: 0x7fdbdceb5190> is getting an implicitly transparent titlebar. This will break when linking against newer SDKs. Use NSWindow's -titlebarAppearsTransparent=YES instead.
Exec path not found (or accessible as executable): /Applications/Keybase.app/Contents/SharedSupport/bin/keybase_

The result of running this is that it launches the client, which indicates "Loading..." and never progresses.

**Comment by jpuerma** [comment]
Hi! I have the same problem. Did anyone fix it? Thanks!

**Comment by zJeremiah** [comment]
I keep getting this error as well, over and over. 
![screen shot 2019-01-18 at 1 02 08 pm](https://user-images.githubusercontent.com/3653633/51410296-6b534200-1b21-11e9-8285-7e9dda74b570.png)

**Comment by Higgins723** [comment]
Same error as above 
![image](https://user-images.githubusercontent.com/5910878/51410532-2976cb80-1b22-11e9-9e54-b8fcbc42ca54.png)

**Comment by maxtaco** [comment]
@Higgins723  and @zJeremiah  did you guys send logs?

**Comment by jbsmith7741** [comment]
I've got the same error as well.
![image](https://user-images.githubusercontent.com/6434838/51616727-ef257980-1ee7-11e9-91cc-994450b435c2.png)

**Comment by spuder** [comment]
I also can't install keybase, when I attempt to send in logs I get a segfault 11

```
keybase log send
Segmentation fault: 11
```
